integration/cloudbuild/docker-compose-mysql.yaml

```diff
@@ -2,17 +2,14 @@ version: '3.1'
 
 services:
   db:
-    image: mysql:5.7
+    image: ${MYSQLD_IMAGE:-mysql:5.7}
     restart: always
     environment:
       MYSQL_ROOT_PASSWORD: 'bananas'
       MYSQL_USER: 'test'
       MYSQL_PASSWORD: 'zaphod'
-      MYSQL_TCP_PORT: ${MYSQL_PORT:-3306}
-    ports:
-      - '${MYSQL_PORT:-3306}:${MYSQL_PORT:-3306}'
```

Comment:
Exporting the ports to the host wasn't necessary. Since the containers join the same "cloudbuild" network, they can talk directly to each other via the container ports.